README.md

```diff
@@ -21,6 +21,25 @@ Commit messages start with the following identifiers enclosed in brackets:
 * feature/liquibase - configuration of the Liquibase structures
 * feature/intest - configuration of the separate module for Integration Tests
 
+
+### 2. Database initialization
+When running the app for the first time, prepare the database to work with.
+Run the SQL statements included in 'database/src/main/resources/init/users.sql', use a user with the DBA role.
+
+### 3. Liquibase changes
+Run the following command to update changes in your database structures:
+```shell
+./gradlew update
+```
+SQL files for step '0' in 'database/src/main/resources/changes' intentionally left blank.
```

Comment:
What is the reason?

Reply:
To serve as a template for the user. Sample sql files are necessary to show how to include them in the 'changelog.yml' while the example app does not have any business logic.